packages/itwin/reports-config-widget/src/test/Reports.test.tsx

```diff
@@ -208,29 +156,16 @@ describe("Reports View", () => {
 
     await user.click(deleteButton);
 
-    await waitForElementToBeRemoved(() =>
-      screen.getByTestId(/rcw-loading-delete/i)
-    );
     await waitForElementToBeRemoved(() => screen.getByRole("dialog"));
 
-    // Should be one less report
-    expect(screen.getAllByTestId("horizontal-tile")).toHaveLength(
-      mockedReportsOriginalLength - 1
-    );
+    expect(mockDeleteReport).toBeCalled();
+    expect(mockGetReports).toBeCalledTimes(2);
```

Comment:
Why do we expect to call `mockGetReports` twice?

Reply:
When it first loads, then after the deletion happens. It should refresh the list which is the second time. Left a comment.